parser.ts

```diff
@@ -19,7 +19,7 @@ import {
   Location,
 } from "./ast";
 
-import { NUM, BOOL, NONE, CLASS, isTagged, STRING, LIST } from "./utils";
+import { NUM, BOOL, NONE, CLASS, isTagged, STRING, LIST, stringify } from "./utils";
```

Comment:
Don't think stringify is used

Reply:
oops you're right; I think I accidentally added it after mistyping string.